SSA/Projects/InstCombine/scripts/test/minimal.lean

```diff
@@ -0,0 +1,102 @@
+import SSA.Projects.InstCombine.LLVM.Semantics
```

Comment:
This file seems unrelated. Also, do you need to update the test cases for this specific change?

Reply:
Oh, yes I didn't mean to have this file still there

Reply:
As for updating the test cases, I wanted to do some edits to the generators first